extensions/jgit/deployment/src/main/java/io/quarkus/jgit/runtime/deployment/JGitProcessor.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.jgit.runtime.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.substrate.ReflectiveClassBuildItem;
+import io.quarkus.deployment.builditem.substrate.RuntimeInitializedClassBuildItem;
+import io.quarkus.deployment.builditem.substrate.SubstrateResourceBundleBuildItem;
+
+class JGitProcessor {
+
+    private static final String FEATURE = "jgit";
```

Comment:
We will change it at some point but for now let's be consistent and add this to `FeatureBuildItem`. Please respect the alphabetical order there.

Reply:
Will do, that's what the `mvn quarkus:create-extension` outputs. Thanks for catching that!

Reply:
Yeah, at some point, I think we will probably move them to each extension but better do it for all the extensions at once.